libs/langchain/langchain/callbacks/mlflow_callback.py

```diff
@@ -127,22 +128,27 @@ class MlflowLogger:
 
     def __init__(self, **kwargs: Any):
         self.mlflow = import_mlflow()
-        tracking_uri = get_from_dict_or_env(
-            kwargs, "tracking_uri", "MLFLOW_TRACKING_URI", ""
-        )
-        self.mlflow.set_tracking_uri(tracking_uri)
+        if "DATABRICKS_RUNTIME_VERSION" in os.environ:
+            self.mlflow.set_tracking_uri("databricks")
+            self.mlf_expid = self.mlflow.tracking.fluent._get_experiment_id()
```

Comment:
@liangz1, I found this [approach](https://github.com/mlflow/mlflow/blob/c9397e6ac007a306b2c99f7c5dc82ebf98651780/mlflow/tracking/fluent.py#L1802) in the mlflow docs for getting the current exp_id. Do you think this is reasonable?